Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/AppCenterContentPage.xaml.cs

```diff
@@ -30,8 +30,8 @@ async void UpdateEnabled(object sender, ToggledEventArgs e)
         void UserIdCompleted(object sender, EventArgs e)
         {
             //TODO: Enable this once NuGet package is updated as part of release process.
```

Comment:
Please remove the TODO comment since the described task is done with the changes.